app/main.py

```diff
@@ -1,8 +1,22 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    peoples_lst = [Person(person["name"], person["age"]) for person in people]
+
+    for person in people:
+        if "wife" in person and person.get("wife") is not None:
+            Person.people[person.get("name")].wife = \
+                Person.people[person.get("wife")]
+        if "husband" in person and person.get("husband") is not None:
+            Person.people[person.get("name")].husband = \
+                Person.people[person.get("husband")]
```

Comment:
no need to use "wife/husband" condition here, u already check this using get method, 
also try not to use line breakers, u can make separate variable for this Person.people[person["name"]] to avoid line breaking, or split line by [] brackets